src/pe_reports/pages.py

```diff
@@ -88,42 +55,76 @@ def credential(chevron_dict, trending_start_date, start_date, end_date, org_uid)
         y_label,
     )
     cred_date_chart.line_chart()
-    breach_table = buildTable(Credential.breach_details(), ["table"])
 
     creds_dict = {
         "breach": Credential.breaches(),
         "creds": Credential.total(),
         "pw_creds": Credential.password(),
-        "breach_table": breach_table,
-        "breachAppendix": buildAppendixList(Credential.breach_appendix()),
+        "breach_table": Credential.breach_details()[:15],
```

Comment:
Is `15` a magic number here, or are you truncating a string for some reason?

Reply:
We only want to display the first 15 rows of the dataframe in the report, so this may be considered a magic number. Should I create a variable called table_rows and pass it there?

Reply:
Addressed in the following commit: [532ff4d](https://github.com/cisagov/pe-reports/pull/565/commits/532ff4d1f303fdb620238bf983d7b3a3738e25b7)